dashboard/app/controllers/musiclab_controller.rb

```diff
@@ -1,12 +1,38 @@
 class MusiclabController < ApplicationController
   ANALYTICS_KEY = CDO.amplitude_api_key
+  PROJECT_BEATS_LEVEL_NAME = 'projectbeats'
 
   def index
     view_options(no_footer: true, full_width: true)
     @body_classes = "music-black"
 
     projects = Projects.new(get_storage_id)
-    @channel_id = projects.most_recent_project_type('music') || ChannelToken.create_channel(request.ip, projects, type: 'music')
+
+    # Find the project ID (channel ID) for the /projectbeats Music Lab project. If there already
+    # is a channel ID with the 'projectbeats' level name, then use that. If not, find the most
+    # recent Music Lab project. If this project is standalone, it should be the existing
+    # /projectbeats project. Create a new project with the 'projectbeats' level name, and copy
+    # source code from the existing project so that users don't lose their code.
```

Comment:
Longer term we could remove the backwards-compatible support?

Reply:
Yep, and I'd also imagine longer term that /projectbeats would just redirect to the standalone level (projects/music), so we wouldn't need to continue supporting this?

Reply:
Oh, just thought of this in another discussion: should we remove the local storage once we've copied code out of it?

Reply:
We can yeah - though this is only copying code if there's an existing project in the projects system, with code on S3, so we're not looking at local storage anyway. Since we changed over to S3 storage, anyone who's used music lab since then should have a project in the projects system.